src/elements/content-sidebar/SidebarNav.js

```diff
@@ -42,10 +42,11 @@ const SidebarNav = ({
     hasDetails,
     hasMetadata,
     hasSkills,
+    intl,
     isOpen,
     onNavigate,
 }: Props) => (
-    <nav>
+    <div aria-label={intl.formatMessage(messages.sidebarNavLabel)} role="tablist">
```

Comment:
Have you tried this if you want to avoid adding in the intl hoc

```jsx
<span className="accessibility-hidden">
    <FormattedMessage {...messages. sidebarNavLabel} />
</span>
```

Reply:
Oh cool, I didn't realize there was something like this. Just tried this out and it seems to work the same. Do we have a preference on avoiding another hoc? It seems like adding the aria-label seems more semantically correct

Reply:
According to this https://accessible360.com/accessible360-blog/use-aria-label-screen-reader-text/ it seems its better to use hidden text just incase screen reader does not support ARIA. Not sure whats better since I have seen both being used at box

Reply:
Interesting, a screen reader not supporting ARIA seems like a paradox 😆 . I'll change it to use the hidden text then